### DISCUSSION TITLE
how data Augmentation for recognition

### DISCUSSION THREAD

**Discussion by Aminfaraji** [PaddlePaddle/PaddleOCR#13723] [Q&A]
Hello, I want to perform data augmentation during the training of the paddleocr model, but with a higher probability. How can I apply these changes to the config file?

this is file config 

`Global:
  use_gpu: true
  epoch_num: 1000
  log_smooth_window: 1
  print_batch_step: 30
  save_model_dir: 
  save_epoch_step: 999
  eval_batch_step:
  - 0
  - 867
  cal_metric_during_train: true
  pretrained_model: 
  checkpoints: 
  save_inference_dir: null
  use_visualdl: false
  infer_img: null
  character_dict_path:
  character_type: fa
  max_text_length: 25
  infer_mode: false
  use_space_char: false
  save_res_path: 
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.99
  clip_norm: 20.0
  lr:
    name: Piecewise
    decay_epochs:
    - 6
    values:
    - 0.0001
    - 1.0e-05
  regularizer:
    name: L2
    factor: 0.0
Architecture:
  model_type: rec
  algorithm: ABINet
  in_channels: 3
  Transform: null
  Backbone:
    name: ResNet45
  Head:
    name: ABINetHead
    use_lang: true
    iter_size: 3
Loss:
  name: CELoss
  smoothing: false
  ignore_index: 100
PostProcess:
  name: ABINetLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
Train:
  dataset:
    name: SimpleDataSet
    data_dir: 
    label_file_list:
    - 
    transforms:
    - DecodeImage:
        img_mode: RGB
        channel_first: false
    - ABINetRecAug: null
    - ABINetLabelEncode: null
    - ABINetRecResizeImg:
        image_shape:
        - 3
        - 32
        - 128
        padding: false
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  loader:
    shuffle: true
    batch_size_per_card: 64
    drop_last: true
    num_workers: 8
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: 
    label_file_list:
    - 
    transforms:
    - DecodeImage:
        img_mode: RGB
        channel_first: false
    - ABINetLabelEncode: null
    - ABINetRecResizeImg:
        image_shape:
        - 3
        - 32
        - 128
        padding: false
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 64
    num_workers: 4
    use_shared_memory: false
profiler_options: null`
